STANDARDS.md

```diff
@@ -32,13 +32,34 @@ Make effective use of namespacing since it will allow co-existence of ambiguous
 
 ### Code Styling
 
-There are two formatting plugins used currently:
+There are two code styling plugins used currently:
 1. Spotless - Provides code formatting and style checking
 2. CheckStyle - Provides style checking
-The advantage of using Spotless over CheckStyle is in addition to check the formatting of the code it also has apply goal that fixes all the style and formatting.
 
-Format Java files with the Eclipse JDT (Java Development Tools) formatter, using the [Spotless Gradle](https://github.com/diffplug/spotless/tree/master/plugin-gradle) plugin. Sample formatting rules can be found [here](https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/formatterConfig.xml). 
+OpenSearch Core had two code styling plugins Spotless and CheckStyle. Spotless itself can format the code and check the standard java code using spotlessApply and spotlessJavaCheck respectively. Both the plugins have different configuration for formatting which restricted us to run Spotless on all the subprojects. Removing CheckStyle helped to run a unified formatting on the codebase. 
 
+How plugins were using CheckStyle?
+
+Plugins were using CheckStyle formatting plugin present in OpenSearch Core’s build.gradle. When core decided to go with Spotless and removed CheckStyle plugin, it caused issue on plugins side and the build broke as CheckStyle plugin was not available to format the code on plugin side. 
```

Comment:
Time in context does not help.
Could you list down versions of OpenSearch where checkstyle is deprecated ?
And also link to the issue, it would help plugin developers understand why opensearch made the change.

Reply:
Added the version of OpenSearch in which the changes were made and linked the issue.